.github/workflows/release.yml

```diff
@@ -0,0 +1,62 @@
+name: Release packages to npm
+
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: googleapis/release-please-action@v4
+        id: release
+        with:
+          target-branch: main
+          config-file: release-please-config.json
+          manifest-file: .release-please-manifest.json
+
+      - uses: actions/checkout@v3
+        # ensure that a publication only occurs when a new release is created:
+        if: ${{ steps.release.outputs.releases_created }}
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 24
+        # ensure that a publication only occurs when a new release is created:
+        if: ${{ steps.release.outputs.releases_created }}
+      - uses: actions/cache@v3
+        with:
+          path: |
+            **/node_modules
+            .yarn/cache
+            .yarn/install-state.gz
+          key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock', '.yarnrc.yml') }}
+        # ensure that a publication only occurs when a new release is created:
+        if: ${{ steps.release.outputs.releases_created }}
+      - run: yarn
+        # ensure that a publication only occurs when a new release is created:
+        if: ${{ steps.release.outputs.releases_created }}
+      - name: Build packages
+        run: |
+          bun run build:evm
+          bun run build:deployments
+          bun run build:sdk
```

Comment:
I think this can just be `yarn build`. Turbo makes sure everything is built in the correct order, and if new packages join the build, we don't have to adjust this here.

Reply:
wanted to avoid building the next.js apps (app and doc) here. those take by far most time

Reply:
I did rename their build steps to `dist` to avoid exactly that ;) So a regular `build` will only build the libs portion of this repo and not the stuff that's being deployed.